include/pmacc/particles/memory/boxes/ExchangePushDataBox.hpp

```diff
@@ -40,15 +40,22 @@ namespace pmacc
     public:
         using PushType = ExchangeMemoryIndex<TYPE, DIM>;
 
+        /**
+         *
+         * @param data particle data storage
+         * @param particleCount current fill level of data
+         * @param maxSize max capacity of data
+         * @param indexTabel index tabel which holds meta data to link particles in data to supercells
```

Comment:
```suggestion
         * @param indexTable index table which holds meta data to link particles in data to supercells
```
Of course, change everywhere.